Dockerfile

```diff
@@ -4,10 +4,13 @@ EXPOSE 4567
 
 WORKDIR /tmp
 COPY Gemfile /tmp
+COPY Gemfile.lock /tmp
 
 RUN gem install bundler && bundle install
 
 RUN mkdir /app
 WORKDIR /app
+RUN rake db:migrate
+RUN rake
```

Comment:
there is no default rake task, this line won't do anything other than error

Reply:
@Williams-Dan that's true. It should be rake run.